newapi/ooniapi/citizenlab.py

```diff
@@ -282,12 +282,19 @@ def add(self, username, cc, new_entry, comment):
 
             self.set_state(username, "IN_PROGRESS")
 
-    def update(self, username, cc, old_entry, new_entry, comment):
+    def update(self, username: str, cc: str, old_entry: dict, new_entry: dict, comment):
+        ks = ("url", "category_code", "category_desc", "date_str", "user", "notes")
+        if old_entry:
+            old_entry_li = [old_entry[k] for k in ks]
+        if new_entry:
+            new_entry_li = [new_entry[k] for k in ks]
         log.debug("updating existing entry")
         cc = cc.lower()
         if len(cc) != 2:
             raise Exception("Invalid country code")
 
+        #entry["category_desc"] = CATEGORY_CODES[entry["category_code"]]
```

Comment:
I think this is a good idea to ensure they are always consistent.

Reply:
Maybe we should do the same for `date_str`

Reply:
We discussed deduplicating this field.

Reply:
Yes I agree with setting it in the backend, I am saying that maybe we can also rely on the backend for stamping the time of when a URL was added/edited.

Reply:
Added a TODO